CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 * For MySQL users, a massive scalability improvement via batched DB writing of internal metadata events. Note that one must add `rewriteBatchedStatements=true` to their JDBC URL in their config in order to take advantage of this
 * In some cases the SFS backend, used for Local, SGE, etc., coerced `WdlFile` to `WdlString` by using `.toUri`. This
 resulted in strings prepended with `file:///path/to/file`. Now absolute file paths will not contain the uri scheme.
+* Added docker hash lookup. Docker images without an explicit digest value will disqualify the task for call caching. See https://github.com/broadinstitute/cromwell#call-caching-docker-tags for more details
```

Comment:
I feel like it adds extra robustness as it allows us to be able to guarantee to the user (provided that we can get the hash), that their job ran with a specific docker image. Regardless of call caching. But you're right 99.9% won't care if they don't use call caching. `Added docker hash lookup for call caching` makes it sounds like it only happens if call caching is enabled which is not true. But I can rephrase it to make it clear that unless you use call caching you probably won't care.